pytealutils/storage/storage.py

```diff
@@ -0,0 +1,29 @@
+from pyteal import App, TealType, Expr, Assert, Seq, Subroutine, If, Int
+
+
+@Subroutine(TealType.anytype)
+def must_get(key: TealType.bytes) -> Expr:
+    """Returns the result of a global storage MaybeValue if it exists, else Assert and fail the program"""
+    maybe = App.globalGetEx(Int(0), key)
+    return Seq(maybe, Assert(maybe.hasValue()), maybe.value())
```

Comment:
👍

Reply:
I don't think **global** should be preferred over **local**. So I recommend you use the `global_` prefix on these methods simiilarly to how you use the `local_` prefix below.